llvm/lib/Target/AIE/AIEBaseInstrInfo.cpp

```diff
@@ -528,14 +528,11 @@ static bool isPreRA(const MachineFunction &MF) {
 }
 
 // Reminder: this is used to compute the latency of RAW edges only.
-std::optional<unsigned>
-AIEBaseInstrInfo::getOperandLatency(const InstrItineraryData *ItinData,
-                                        const MachineInstr &DefMI,
-                                        unsigned DefIdx,
-                                        const MachineInstr &UseMI,
-                                        unsigned UseIdx) const {
-  std::optional<int> Lat = getSignedOperandLatency(
-          ItinData, DefMI, DefIdx, UseMI, UseIdx, SDep::Data);
```

Comment:
Is there a way not to commit these un-necessary diff lines?

Reply:
Yes, have cleaned up.